pages/templates/partials/past_results.html

```diff
@@ -36,31 +36,25 @@ <h2 class="mb20">
           </a>
           {% endif %}
 
-          <hr>
+          {% comment %}Wagtail places HTML tags in blank RichTextFields, so a
+          blank RichTextField evaluates as a string instead of None. We need
+          to strip tags to make sure they properly evaluate as None and so we
+          don't put linebreaks on the page when empty. This is fixed in Wagtail
+          2.2.2.
 
-          <h2>Nominate a Candidate</h2>
-          <p class="fs18">
-            Our Revolution is made up of hundreds of local groups all across the world,
-            and all of our endorsements must first originate with a nomination from an
-            official group. <strong>Get started submitting your group’s nomination now.
-            </strong>
-          </p>
+          See: https://github.com/wagtail/wagtail/issues/2336
+          {% endcomment %}
+          {% if page.secondary_copy|striptags %}
+          <hr>
+          {{ page.secondary_copy|richtext }}
 
+          {% if page.button_show %}
           <a href="/groups/nominations"
```

Comment:
Might be out of scope but should use template tag or something like that rather than hardcode url into template.